util/config/find-python.sh

```diff
@@ -5,16 +5,17 @@
 check_python() {
   if command -v $1 > /dev/null 2>&1
   then
-    if $1 --version > /dev/null 2>&1
+    PYTHON_VERSION_OUTPUT=$($1 --version 2>&1)
+    if [ $? -eq 0 ]
     then
       MIN_MINOR_PYTHON_VERSION=5
-      if $1 -c "import sys; sys.exit(int(sys.version_info[:2] >= (3, $MIN_MINOR_PYTHON_VERSION)))" >/dev/null 2>&1
+      if printf "Python 3.%s\n%s\n" "$MIN_MINOR_PYTHON_VERSION" "$PYTHON_VERSION_OUTPUT" | sort -V -C
```

Comment:
I have a slight nit that you use `--version-sort --check` instead of `-V -C` to make this more immediately readable, as I had to go read the man page to understand this

Reply:
I was hoping to allude to "__V__enture __C__apital" 😄 

Just kidding. I'll make the change.